go/vault/api/policy.go

```diff
@@ -0,0 +1,88 @@
+package api
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"github.com/oasisprotocol/oasis-core/go/common/quantity"
+	"github.com/oasisprotocol/oasis-core/go/staking/api/token"
+)
+
+// AddressState is the state stored for the given address.
+type AddressState struct {
+	// WithdrawPolicy is the active withdraw policy.
+	WithdrawPolicy WithdrawPolicy `json:"withdraw_policy"`
```

Comment:
Shouldn't this policy be applied to the vault?

Reply:
What do you mean? This is a per-address policy for the given vault.

Reply:
Do we want to have a similar policy for vault? So how much can all addresses withdraw together in some time period.

Reply:
Let's keep it simple for now.

Reply:
I would say that having a policy for a vault is simpler than having a policy for every address that can withdraw from it.
But yeah, we can keep it as it is.

Reply:
That's true, but we do want to have a list of addresses that are allowed to withdraw anything at all, so per-address state is needed anyway.